install.sh

```diff
@@ -12,7 +12,7 @@ then
 elif [ "$OSTYPE" == "Linux" ];
 then
      LINUX_DIST=$(cat /etc/os-release | grep "^NAME=" | cut -d '=' -f2 | tr -d '"')
-     if [[ $LINUX_DIST =~ "Debian" ]]
+     if [[ $LINUX_DIST =~ "Debian" ]] || [[ $LINUX_DIST =~ "Ubuntu" ]]
```

Comment:
These can be combined as `if [[ $LINUX_DIST =~ "Debian" || $LINUX_DIST =~ "Ubuntu" ]]`

Reply:
Yes that's perfectly fine